vmm/src/cpu.rs

```diff
@@ -1784,6 +1784,22 @@ impl CpuManager {
     pub(crate) fn vcpus_kill_signalled(&self) -> &Arc<AtomicBool> {
         &self.vcpus_kill_signalled
     }
+    pub fn get_cpuid_leaf(
```

Comment:
Nit: `pub(crate)` should be enough and also need a whiteline above

Reply:
Looks like this is a mostly wrapper function and only used once. I think you should just remove this wrapper function and call into `get_cpuid_values` directly.

Reply:
get_cpuid_values is vcpu specific function and vcpu is private property of CpuManager, How do I exactly remove the wrapper here?